custom-deployment/action.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Custom Deployment'
+description: 'Deploy action for VIP applications'
+inputs:
+  WPVIP_DEPLOY_TOKEN:
+    description: 'The deploy token for WPVIP'
+    required: true
+    default: ''
+  ENVIRONMENT_SLUG:
+    description: 'The environment slug for deployment'
+    required: true
+    default: ''
+  RELEASE_NAME:
+    description: 'The name of the release'
+    required: true
+    default: ''
```

Comment:
Minor: I think we can omit the default value since the inputs are required: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault